### PR TITLE
Separate jobs

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Separated into 3 dependent stages

### CODE REVIEW

File: .github/workflows/build.yml
Comment (atymchuk) [inline .github/workflows/build.yml:6]: Is this branch long-lived? Cause if you delete it, the workflow will never run.

File: .github/workflows/build.yml
Comment (atymchuk) [inline .github/workflows/build.yml:38]:
```suggestion
        uses: percona-platform/setup-node@v2
```
We upgraded the script, it should now be just v2.

File: .github/workflows/build.yml
Comment (atymchuk) [inline .github/workflows/build.yml:52]: Not sure we need this as a separate step. We could continue running steps in the same job. Whenever a step fails, the whole job fails. Unless you want to continue despite a failing job, this is probably not necessary.

File: .github/workflows/build.yml
Comment (atymchuk) [inline .github/workflows/build.yml:63]: Same as above.

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml
Comment (PaulJacobs-percona) [reply .github/workflows/build.yml:6]: Right, it was just for testing, I intended fixing it after merge

File: .github/workflows/build.yml
Comment (PaulJacobs-percona) [reply .github/workflows/build.yml:52]: The whole point of this PR was to separate into jobs. There was no other change. The reason is that you can't see what stage has failed without drilling down. With jobs, you see immediately which part of setup/build/(spell/grammar) check has a problem.

Comment (atymchuk) [reply .github/workflows/build.yml:52]: It's an overkill, since every job starts a fresh operating system, which slows down the workflow and burns the resources. 

You can see what stage has failed without drilling down. Example attached.

Screenshot 2021-02-12 at 12 06 24

But I leave it up to you :)

**Review by atymchuk — APPROVED** [review]

**Comment by PaulJacobs-percona** [comment]
Convinced not a good way to see separate stage statuses.

**Pull request closed without merging**
